cmd/semantic-release/semantic-release.go

```diff
@@ -67,15 +70,26 @@ func main() {
 	case "up":
 		logger.Info(colorYellow + "\nSemantic Version just started the process...\n\n" + colorReset)
 
-		semantic := newSemantic(logger, upgradeVersionCmd, gitHost, groupName, projectName, username, password, upgradePyFile)
-
-		if err := semantic.GenerateNewRelease(); err != nil {
-			logger.Error(err.Error())
-			os.Exit(1)
+		semantic := newSemantic(logger, upgradeVersionCmd, gitHost, groupName, projectName, username, password, upgradePyFile, branchName)
+
+		if *commitLint {
+			if *branchName == "" {
+				logger.Error(colorRed + "\nThe argument -branch-name must be set when --commit-lint is true.\n\n" + colorReset)
+			}
+
+			logger.Info(colorYellow + "\nSemantic Version commit lint started...\n\n" + colorReset)
+			err := semantic.CommitLint()
+			if err != nil {
+				os.Exit(1)
```

Comment:
@esequielvirtuoso should we log the error message?

Reply:
@ebbauer 
We're logging it at the `semantic` package.